applications/plot_layout_array.py

```diff
@@ -0,0 +1,182 @@
+    layout_array_name (str, optional)
+        Name of the layout array.
+    rotate_angle (float, optional)
+        Angle to rotate the array before plotting (in degrees).
+    show_tel_label (bool, optional)
+        Shows the telescope labels in the plot.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    .. code-block:: console
+
+        python applications/plot_layout_array.py --figure_name northern_array_alpha \
+        --layout_array_name test_layout
+
+"""
+
+import logging
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from astropy import units as u
+
+import simtools.util.general as gen
+from simtools import io_handler
+from simtools.configuration import configurator
+from simtools.layout.layout_array import LayoutArray
+from simtools.visualization.visualize import plot_array
+
+
+def _parse(label, description, usage):
+    """
+    Parse command line configuration
+
+    Parameters
+    ----------
+    label: str
+        Label describing application.
+    description: str
+        Description of application.
+    usage: str
+        Example on how to use the application.
+
+    Returns
+    -------
+    CommandLineParser
+        Command line parser object
+
+    """
+    config = configurator.Configurator(label=label, description=description, usage=usage)
+
+    config.parser.add_argument(
+        "--figure_name",
+        help="Name of the output figure to be saved into as a pdf.",
+        type=str,
+        required=False,
+        default=None,
+    )
+    config.parser.add_argument(
+        "--rotate_angle",
+        help="Angle to rotate the array before plotting (in degrees).",
+        type=str,
+        nargs="+",
+        required=False,
+        default=None,
+    )
+    config.parser.add_argument(
+        "--show_tel_label",
+        help="Shows the telescope labels in the plot.",
+        action="store_true",
+        required=False,
+        default=False,
+    )
+    input_group = config.parser.add_mutually_exclusive_group()
+    input_group.add_argument(
+        "--telescope_list",
+        help="ECSV file with the list of telescopes",
+        nargs="+",
+        type=str,
+        required=False,
+        default=None,
+    )
+    input_group.add_argument(
+        "--layout_array_name",
+        help="Name of the layout array.",
+        nargs="+",
+        type=str,
+        required=False,
+        default=None,
+    )
+
+    return config.initialize(db_config=True)
+
+
+def main():
+
+    label = Path(__file__).stem
+    description = "Plots layout array."
+    usage = "python applications/plot_layout_array.py --layout_array_name test_layout"
+    args_dict, _ = _parse(label, description, usage)
+    io_handler_instance = io_handler.IOHandler()
+
+    if args_dict["rotate_angle"] is None:
+        rotate_angles = [0 * u.deg]
+    else:
+        rotate_angles = []
+        for _, one_angle in enumerate(args_dict["rotate_angle"]):
```

Comment:
Why do you use `enumerate` without actually using the counter? Seems unnecessary.

Reply:
I thought codacy would complain but you are right, since I am using only the value and not the counter I dont need enumerate. I am removing it now